package/boot/uboot-envtools/files/mediatek_filogic

```diff
@@ -93,7 +94,8 @@ ubnt,unifi-6-plus)
 	ubootenv_add_uci_config "/dev/mtd1" "0x0" "0x80000" "0x10000"
 	;;
 xiaomi,mi-router-wr30u-112m-nmbm|\
-xiaomi,mi-router-wr30u-stock|\
+xiaomi,mi-router-wr30u-stock|\             
```

Comment:
the whitepsaces at the end should be removed

Reply:
Thanx. Fixed.